doc/TEMPLATES.md

```diff
@@ -291,6 +291,49 @@ html`<div>${fragment}</div>`;
 // Error: Unexpected child element count of zero for given DocumentFragment.
 ```
 
+## Supported native tags
```

Comment:
As requested, added a section for `#supported-native-tags` in our `TEMPLATES.md` file (so that we can link to it from our error messaging).